codepropertygraph/src/main/resources/schemas/base.json

```diff
@@ -89,10 +89,10 @@
            {"edgeName": "AST",
             "inNodes": [
               {"name": "METHOD_RETURN", "cardinality":"1:1"},
-              {"name": "METHOD_PARAMETER_IN"},
-              {"name": "MODIFIER"},
+              {"name": "METHOD_PARAMETER_IN", "cardinaliy": "n:1"},
```

Comment:
I think you mixed up the ordering:
A METHOD can connect via AST to n METHOD_PARAMETR_IN
and
A METHOD_PARAMETER_IN can have 1 incoming AST from METHOD.

But you are stating it the other way around. Look at the IDENTIFIER to LOCAL via REF relation as a correct example.

Reply:
thanks for paying attention. 
maybe this was just a test? you passed it either way. 
will review all and let you know

Reply:
yup, they all need to be reversed...